src/mlpack/core/math/ccov.hpp

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file ccov.hpp
+ * @author Ryan Curtin
+ * @author Conrad Sanderson
+ *
+ * ColumnCovariance(X) is same as cov(trans(X)) but without the cost of computing trans(X)
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_MATH_CCOV_HPP
+#define MLPACK_CORE_MATH_CCOV_HPP
+
+#include <mlpack/core.hpp>
+#include <mlpack/core/util/log.hpp>
+
+namespace mlpack {
+namespace math /** Miscellaneous math routines. */ {
+
+template<typename eT>
+inline
+arma::Mat<eT>
+ColumnCovariance(const arma::Mat<eT>& A, const arma::size_t norm_type = 0)
```

Comment:
`std::size_t`

Reply:
@conradsnicta , Done with the correction, I hope I haven't made any more errors.